scripts/builtin/_genBuiltinDocs.py

```diff
@@ -147,10 +147,10 @@ def parseInformation(markDownArray, fileName):
 
     for line in markDownArray:
         if INPUT:
-            inputParameterCount, INPUT, INPUT_FINISHED, I_HEADER_MISSING, commentLineCount = parseParam(inputParameters, inputParameterCount, INPUT, INPUT_FINISHED, I_HEADER_MISSING, commentLineCount, additionalInfos, line)
+            inputParameterCount, INPUT, INPUT_FINISHED, I_HEADER_MISSING, commentLineCount = parseParam(inputParameters, inputParameterCount, INPUT, INPUT_FINISHED, I_HEADER_MISSING, commentLineCount, additionalInfos, line, OUTPUT)
```

Comment:
This script needs to be well tested 🥇 . For now, we can use this only 1 or 2 scripts in `scripts/staging` folder. After which, we can apply generically. 😃